[20160628-104918]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipaadm', URI='/calipaadm/GravarTermo.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160628-104918]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160628-104918]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160628-104918]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160628-104918]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160628-104918]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160628-104918]: 	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
[20160628-104918]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:624)
[20160628-104918]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160628-104918]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160628-104918]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-104918]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160628-104918]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160628-104918]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-104918]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160628-104918]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160628-104918]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160628-104918]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160628-104918]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160628-104918]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160628-104918]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160628-104918]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160628-104918]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160628-104918]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160628-104918]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160628-104918]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160628-104918]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160628-104918]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160628-104918]: 	at java.lang.Thread.run(Thread.java:662)
[20160628-104918]: Caused by: java.lang.NullPointerException
[20160628-104918]: 	at java.io.File.<init>(File.java:222)
[20160628-104918]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.GravarTermoAction.executeAction(GravarTermoAction.java:46)
[20160628-104918]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.AdministracaoBaseAction.execute(AdministracaoBaseAction.java:36)
[20160628-104918]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160628-104918]: 	... 26 more
[20160628-104918]: .
[20160628-104924]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipaadm', URI='/calipaadm/GravarTermo.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160628-104924]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160628-104924]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160628-104924]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160628-104924]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160628-104924]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160628-104924]: 	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
[20160628-104924]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:624)
[20160628-104924]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160628-104924]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160628-104924]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-104924]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160628-104924]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160628-104924]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-104924]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160628-104924]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160628-104924]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160628-104924]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160628-104924]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160628-104924]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160628-104924]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160628-104924]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160628-104924]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160628-104924]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160628-104924]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160628-104924]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160628-104924]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160628-104924]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160628-104924]: 	at java.lang.Thread.run(Thread.java:662)
[20160628-104924]: Caused by: java.lang.NullPointerException
[20160628-104924]: 	at java.io.File.<init>(File.java:222)
[20160628-104924]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.GravarTermoAction.executeAction(GravarTermoAction.java:46)
[20160628-104924]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.AdministracaoBaseAction.execute(AdministracaoBaseAction.java:36)
[20160628-104924]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160628-104924]: 	... 26 more
[20160628-104924]: .
[20160628-132812]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipaadm', URI='/calipaadm/GravarTermo.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, fundo.cFundInves='6275', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160628-132812]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160628-132812]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160628-132812]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160628-132812]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160628-132812]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160628-132812]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160628-132812]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160628-132812]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160628-132812]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160628-132812]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-132812]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160628-132812]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160628-132812]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-132812]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160628-132812]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160628-132812]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160628-132812]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160628-132812]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160628-132812]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160628-132812]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160628-132812]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160628-132812]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160628-132812]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160628-132812]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160628-132812]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160628-132812]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160628-132812]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160628-132812]: 	at java.lang.Thread.run(Thread.java:662)
[20160628-132812]: Caused by: java.lang.NullPointerException
[20160628-132812]: 	at java.io.File.<init>(File.java:222)
[20160628-132812]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.GravarTermoAction.executeAction(GravarTermoAction.java:46)
[20160628-132812]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.AdministracaoBaseAction.execute(AdministracaoBaseAction.java:36)
[20160628-132812]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160628-132812]: 	... 26 more
[20160628-132812]: .
[20160628-133014]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipaadm', URI='/calipaadm/GravarTermo.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, fundo.cFundInves='6275', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160628-133014]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160628-133014]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160628-133014]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160628-133014]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160628-133014]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160628-133014]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160628-133014]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160628-133014]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160628-133014]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160628-133014]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-133014]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160628-133014]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160628-133014]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-133014]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160628-133014]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160628-133014]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160628-133014]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160628-133014]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160628-133014]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160628-133014]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160628-133014]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160628-133014]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160628-133014]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160628-133014]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160628-133014]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160628-133014]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160628-133014]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160628-133014]: 	at java.lang.Thread.run(Thread.java:662)
[20160628-133014]: Caused by: java.lang.NullPointerException
[20160628-133014]: 	at java.io.File.<init>(File.java:222)
[20160628-133014]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.GravarTermoAction.executeAction(GravarTermoAction.java:46)
[20160628-133014]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.AdministracaoBaseAction.execute(AdministracaoBaseAction.java:36)
[20160628-133014]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160628-133014]: 	... 26 more
[20160628-133014]: .
[20160628-133027]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipaadm', URI='/calipaadm/GravarTermo.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, fundo.cFundInves='6275', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160628-133027]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160628-133027]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160628-133027]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160628-133027]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160628-133027]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160628-133027]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160628-133027]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160628-133027]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160628-133027]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160628-133027]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-133027]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160628-133027]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160628-133027]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160628-133027]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160628-133027]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160628-133027]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160628-133027]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160628-133027]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160628-133027]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160628-133027]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160628-133027]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160628-133027]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160628-133027]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160628-133027]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160628-133027]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160628-133027]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160628-133027]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160628-133027]: 	at java.lang.Thread.run(Thread.java:662)
[20160628-133027]: Caused by: java.lang.NullPointerException
[20160628-133027]: 	at java.io.File.<init>(File.java:222)
[20160628-133027]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.GravarTermoAction.executeAction(GravarTermoAction.java:46)
[20160628-133027]: 	at br.com.bradesco.custodia.portal.portaladm.apresentacao.AdministracaoBaseAction.execute(AdministracaoBaseAction.java:36)
[20160628-133027]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160628-133027]: 	... 26 more
[20160628-133027]: .
